(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> cannot concatenate 'str' and 'exceptions.AttributeError' objects"
p3
sS'layer'
p4
S'/home/archer/Documents/web2py/applications/baadal/controllers/action.py'
p5
sS'code'
p6
S'import Baadal\nimport json\ndef start():\n    conn = Baadal.Connection("http://10.237.23.178:35357/v2.0", "admin", "admin", "baadal")\n    if conn:\n        vmid = request.vars.vmid\n        vm = conn.findBaadalVM(id=\'e3192a6d-200f-45c0-b5e5-2f9d2008be20\')\n        if vm:\n            vm.start()\n            pass\n        return json.dumps(dict(status=\'success\'))\n    else:\n        return json.dumps(dict(status=\'failure\'))\n\n\nresponse._vars=response._caller(start)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/archer/Documents/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/archer/Documents/web2py/applications/baadal/controllers/action.py", line 16, in <module>\n  File "/home/archer/Documents/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/archer/Documents/web2py/applications/baadal/controllers/action.py", line 7, in start\n    vm = conn.findBaadalVM(id=\'e3192a6d-200f-45c0-b5e5-2f9d2008be20\')\n  File "applications/baadal/modules/Baadal.py", line 375, in findBaadalVM\n    raise BaadalException("No matching VM found" + e)\nTypeError: cannot concatenate \'str\' and \'exceptions.AttributeError\' objects\n'
p11
s.